main/inc/lib/myspace.lib.php

```diff
@@ -990,6 +1007,305 @@ public static function display_user_overview_export_options()
         }
     }
 
+    /**
+     * Export to cvs a list of users who were enrolled in the lessons.
+     * It is necessary that in the extra field, a company is defined.
+     *
+     * @param null $startDate
```

Comment:
The type in the @param documentation should probably be "string|null" rather than null. Although it *can* be null, it only makes sense to have a param defined if we sometimes have a value there...